### PR TITLE
Hide existing non-partner charts in "library" catalog

### PR DESCRIPTION
**Pull request by rayandas**
Fixes: https://github.com/rancher/rancher/issues/35347

### CODE REVIEW

**Review by PennyScissors — CHANGES_REQUESTED** [review]
This repository only contains Helm 2 charts which aren't compatible in the new UI. For that reason, you don't need to set the rancher-version annotation, instead add the rancher min/max to the questions.yaml. If a chart doesn't have a questions.yaml file (some really old charts may not have one), you need to create a questions.yaml file and add the min/max to it in order to hide it. This PR should only contain modifications to question.y*ml files, and all charts should have either a modified or newly added questions.y*ml.

**Comment by rayandas** [comment]
@PennyScissors Got it. I will update this PR today with the suggested changes.

File: charts/longhorn/latest/questions.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+rancher_max_version: 2.6.2
```
Comment (deniseschannon) [inline charts/longhorn/latest/questions.yml:1]: We should allow Longhorn as this is one of the actively maintained charts by SUSE/Rancher.

**Review by deniseschannon — CHANGES_REQUESTED** [review]

File: charts/longhorn/latest/questions.yml (hunk 1 above)
Comment (rayandas) [reply charts/longhorn/latest/questions.yml:1]: alright. I'll remove the rancher_max_version from longhorn.

**Review by PennyScissors — CHANGES_REQUESTED** [review]
The following questions files are missing a rancher_max_version.
```
./charts/etcd-operator/v0.9.0/questions.yml
./charts/vault-operator/v0.1.3/questions.yml
./charts/datadog/v1.33.0/questions.yml
```

**Comment by deniseschannon** [comment]
@rayandas When I add your branch into a v2.6.3-rc1, I see lots of non-partner charts still:

docker-registry
etcd-operator
fluentd-aggregator
grafana
harbor-scanner-trivvy
kubeflow
memcached
openfaas
prometheus
redis

**Comment by rayandas** [comment]
@deniseschannon apologies I missed few of them. I just pushed the fix. these charts shouldn't be visible anymore.
/cc: @PennyScissors

**Review by deniseschannon — APPROVED** [review]

**Review by deniseschannon — APPROVED** [review]

**Pull request merged**